lib/noticed/delivery_methods/ios.rb

```diff
@@ -32,8 +32,12 @@ def deliver
       def format_notification(apn)
         apn.topic = evaluate_option(:bundle_identifier)
 
-        if (method = config[:format])
-          method = event.send(method, apn) if method.is_a?(Symbol) && event.respond_to?(method)
+        method = config[:format]
+        # Call method on Notifier if defined
+        if method&.is_a?(Symbol) && event.respond_to?(method)
+          event.send(method, apn)
+        # If Proc, evaluate it on the Notification
+        elsif method.present?
```

Comment:
What are your thoughts on having this mirror the `evaluate_options` call? , where it checks `method.respond_to(:call)` and then falls back to the string value of method if it exists?  This way the iOS format would match the way `evaluate_options` works for custom delivery methods with the final fallback to `message` if it exists.

Reply:
Yes, that's what I intended actually. 
```suggestion
        elsif method&.respond_to?(:call)
```